scheduler/server/scheduler.go

```diff
@@ -23,9 +23,19 @@ type Scheduler interface {
 	SetSchedulerStatus(maxTasks int) error
 
 	GetSchedulerStatus() (int, int)
+
+	GetClassLoadPcts() (map[string]int32, error)
```

Comment:
We're kind of mixing the idea of replaceable algorithms and fully migrating to the class-based alg. At this point we've basically changed the API to require the use of a class-based alg. Maybe once this is finalized, we don't care and we get rid of the "orig" alg altogether.